src/components/rdpassspray/rdpassspray.tsx

```diff
@@ -0,0 +1,222 @@
+import { Button, Stack, TextInput, Checkbox } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+
+/**
+ * Represents the form values for the RDPassSpray component.
+ */
+interface FormValuesType {
+    targetIP: string;
+    usernameFile: string;
+    passwordFile: string;
+    domain: string;
+    rate: string;
+}
+
+/**
+ * The RDPassSpray component.
+ * @returns The RDPassSpray component.
+ */
+const RDPassSpray = () => {
+    // Component state variables
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution
+    const [allowSave, setAllowSave] = useState(false); // State variable to allow saving of output
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if output has been saved
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+    const [verboseMode, setVerboseMode] = useState(false); // State variable for verbose mode
+
+    // Component Constants
+    const title = "RDPassSpray";
+    const description =
+        "RDPassSpray is a tool for brute-forcing RDP services using username and password lists. It's used for password spraying attacks against remote desktop services.";
+    const steps =
+        "=== Required ===\n" +
+        "Step 1: Input the target IP address or range.\n" +
+        "Step 2: Provide a file containing usernames to test.\n" +
+        "Step 3: Provide a file containing passwords to test.\n" +
+        "=== Optional ===\n" +
+        "Step 4: Input a domain if necessary.\n" +
+        "Step 5: Specify a spray rate to avoid detection (number of attempts per second).\n" +
+        "Step 6: Enable verbose mode for more detailed output.\n";
+    const sourceLink = ""; // Link to the source code
+    const tutorial = ""; // Link to the official documentation/tutorial
+    const dependencies = ["rdpassspray"]; // Contains the dependencies required by the component.
```

Comment:
Can you please check the dependency again? I still cannot find it: 

![image](https://github.com/user-attachments/assets/d7ab7e41-1abd-4c7a-9129-68537c43ce6d)

I am not sure if this has been packaged for Kali.